<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>          &quot;Hazır  oldugunda  Basla'ya  bas !&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>&quot;Basla&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>        &quot;Time : &quot; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRetry">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>&quot;Yeniden Dene &quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDot">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>99</y>
     <width>461</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;  &quot;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameInfo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>&quot;kullanıcı adınızı giriniz : &quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot; &quot;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
